### PR TITLE
CUAV Nora & X7 Pages 

### PR DESCRIPTION
**Pull request by MadRC**
Hi I have added pages for the CUAV Nora and X7 Autopilots. 

I have checked these and tested on a local build but please can you give them a 10 over. 

You may need to squish these as well sorry.

### CODE REVIEW

**Comment by Hwurzburg** [comment]
it would be good if you could include the default UART port mapping, and DSHot groups like the other pages....see MatekF405-Wing for example...it helps reduce the questions....thanks

**Comment by MadRC** [comment]
I’ll see if I can find the info if I get a chance to look at it again.

**Comment by Hwurzburg** [comment]
Ian, why close? I could add the info, if you like....is there some other issue?

File: common/source/docs/common-cuav-X7-overview.rst [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+X7 Pro
+======
+
+The X7 PRO® is an advanced autopilot independently designed by CUAV®. It uses a higher-performance STM32H7 processor and integrates industrial-grade sensors and ultra-low temperature drift sensors. Compared with the previous flight controller,it has better performance and more reliability.
+
+Features 
+========
+
+-  Internal IMU isolation
+-  Modular design, can be custom carrier board
```
Comment (amilcarlucas) [inline common/source/docs/common-cuav-X7-overview.rst:23]: can be mounted on a custom carrier board

File: common/source/docs/common-cuav-X7-overview.rst [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+-  Supports USB_HS
```
Comment (amilcarlucas) [inline common/source/docs/common-cuav-X7-overview.rst:24]: Supports high-speed USB (480Mbit/s)

File: common/source/docs/common-cuav-X7-overview.rst [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+-  Supports more dshot output
```
Comment (amilcarlucas) [inline common/source/docs/common-cuav-X7-overview.rst:25]: up-to 12 dshot outputs

File: common/source/docs/common-cuav-X7-overview.rst [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+-  Tempratture Controled IMU
```
Comment (amilcarlucas) [inline common/source/docs/common-cuav-X7-overview.rst:26]: temperature

File: common/source/docs/common-cuav-X7-overview.rst [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+   -  USB Input: 4.75~5.25V
+   -  High-power servo rail, up to 36V
+      (servo rail does not power the autopilot)
+   -  Dual voltage and current monitor inputs
+   -  CUAV Nora can be triple redundant if power is provided
+      to both battery monitor inputs and the USB port
+
+-  **Interfaces**
+
+   -  14 PWM outputs （12 supports Dshot）
```
Comment (amilcarlucas) [inline common/source/docs/common-cuav-X7-overview.rst:66]: support

File: common/source/docs/common-cuav-nora-overview.rst [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+
+Nora® is an advanced autopilot independently designed by CUAV®. It uses a higher-performance STM32H7 processor and integrates industrial-grade sensors and ultra-low temperature drift sensors. Compared with the previous flight controller,it has better performance and more reliability. This board can run ArduPilot firmware perfectly. Nora is ideal for academic research and commercial systems integration.
+
+Features 
+========
+
+-  Internal IMU isolation
+-  Connection ports on the side of autopilot
+-  Supports USB_HS
```
Comment (amilcarlucas) [inline common/source/docs/common-cuav-nora-overview.rst:19]: Supports high-speed (480Mbit/s) USB

File: common/source/docs/common-cuav-nora-overview.rst [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+-  Supports more dshot output
```
Comment (amilcarlucas) [inline common/source/docs/common-cuav-nora-overview.rst:20]: Supports up-to 12 Dshot outputs

File: common/source/docs/common-cuav-nora-overview.rst [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+-  Tempratture Controled IMU
```
Comment (amilcarlucas) [inline common/source/docs/common-cuav-nora-overview.rst:21]: temperature

File: common/source/docs/common-cuav-nora-overview.rst [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+   -  USB Input: 4.75~5.25V
+   -  High-power servo rail, up to 36V
+      (servo rail does not power the autopilot)
+   -  Dual voltage and current monitor inputs
+   -  CUAV Nora can be triple redundant if power is provided
+      to both battery monitor inputs and the USB port
+
+-  **Interfaces**
+
+   -  14 PWM outputs （12 supports Dshot）
```
Comment (amilcarlucas) [inline common/source/docs/common-cuav-nora-overview.rst:54]: support

**Comment by MadRC** [comment]
Sorry I will look at this all tomorrow was closed as I I seen some issues as shown above.

**Comment by Hwurzburg** [comment]
Ian, Hongle submitted a set of pages also...they appear almost identical.....so I am working off those....I will consolidate the two X7 pages into one like you did, however....do you want to close this PR in that case?

one question is CUAV and Nora and X7 registered trademarks? Hongle's stuff has them as such, but they must actually be registered for us to show the superscript in the docs....

**Comment by MadRC** [comment]
I’m not sure tbh. I’d not got back to this yet to make the changes requested but i will close this now and you can use the official ones.

I built these up manually from the info on the their site and the other CUAV page as a template, I guess they did the same. 

I’ll close this now. 

Thanks.

**Comment by Hwurzburg** [comment]
thanks Ian....sorry for all the time and work lost...in the future I will be sure to check with both you and Hongle....I had thought you were communicating re this ....sorry again...

**Comment by MadRC** [comment]
I just seen it on Tridges list for the beta to be done and thought I’d try and help out. I was not aware they do their own pages.  

No issues.

**Pull request closed without merging**